.github/workflows/ci.yml

```
@@ -0,0 +1,9 @@
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+      steps:
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 5: mapping values are not allowed in this context

(syntax-check)